tests/rbd/rbd_utils.py

```diff
@@ -481,6 +481,8 @@ def image_resize(self, pool_name, image_name, size, **kw):
             for passphrase in kw.get("passphrase"):
                 for key, value in passphrase.items():
                     cmd += f" --{key} {value}"
+            if kw.get("allow_shrink"):
```

Comment:
Sorry if I'm missing something, in line 491 we are adding --allow-shrink
Can't we make use of it (optimise) ?

Reply:
So there are few customized command arguments I am expecting when it is an encrypted image, for example "all", and the passphrase in itself. And for encrypted images I would like to pass --allow-shrink only when it is specified by user. 

Because of these reasons I would like to retain the code as is. Also I am returning the value from line 493 itself when image is encrypted so it won't even reach line 498. 

Please let me know if this explanation suffices or if you have any better way of optimisation.